_posts/2022-03-31-agile-team-natural-habitat.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: post
+title: "An Agile team in its natural habitat"
+author: [michal.kosmulski]
+tags: [tech, agile, testing, "April 1st"]
+---
+In this picture, we can see an Agile team in its natural habitat at Allegro:
+
```

Comment:
It doesn't look good... 🤔 
<img width="654" alt="Screenshot 2022-03-31 at 10 06 23" src="https://user-images.githubusercontent.com/90947/161007765-2bcaad44-e495-49c7-8913-d5d8c32de043.png">

Reply:
Good point - I added a custom excerpt